docs/api/python/rnn.md

```diff
@@ -0,0 +1,339 @@
+# RNN Cell API
+
+```eval_rst
+.. currentmodule:: mxnet.rnn
+```
+
+## Overview
+
+The `rnn` module includes the recurrent neural network (RNN) cell APIs, a suite of tools for building the symbolic graph of RNN.
+
+## The `rnn` module
+
+### Cell interfaces
+
+```eval_rst
+.. autosummary::
+    :nosignatures:
+
+    BaseRNNCell.__call__
+    BaseRNNCell.unroll
+    BaseRNNCell.reset
+    BaseRNNCell.begin_state
+    BaseRNNCell.unpack_weights
+    BaseRNNCell.pack_weights
+```
+
+The cell API operates on symbols and returns output symbols based on the type of RNN. Take
+Long-short term memory (LSTM) as an example:
```

Comment:
Change to:
"When working with the cell API, the precise input and output symbols 
depend on the type of RNN you are using. Take Long Short-Term Memory (LSTM) for example:"